packages/zowe-explorer/__tests__/__unit__/command/unixCommandHandler.unit.test.ts

```diff
@@ -179,6 +179,19 @@ describe("UnixCommand Actions Unit Testing", () => {
                     getBaseProfile: jest.fn(),
                     getDefaultProfile: mockdefaultProfile,
                     validProfile: ValidProfileEnum.VALID,
+                    getProfileInfo: jest.fn().mockReturnValue({
+                        usingTeamConfig: true,
+                        getAllProfiles: jest.fn().mockReturnValue(["dummy"]),
+                        mergeArgsForProfile: jest.fn().mockReturnValue({
+                            knownArgs: [
+                                { argName: "port", argValue: "TEST", secure: false },
+                                { argName: "host", argValue: "TEST", secure: false },
+                                { argName: "user", argValue: "TEST", secure: false },
+                                { argName: "password", argValue: "TEST", secure: false },
+                            ],
+                        }),
+                        loadSecureArg: jest.fn().mockReturnValue("fake"),
+                    } as any),
```

Comment:
I noticed that this mock is duplicated multiple times in this file but with the same return values. Do you mind saving this mock to a constant and referencing it elsewhere to avoid code duplication? 

Having a single mock definition will help with updating tests in the future, in case we would have to update the return values from `getProfileInfo`.

Reply:
Thankyou for this suggestion traeok. Addressed all the comments you have mentioned